docs/components/badges/index.html

```diff
@@ -0,0 +1,66 @@
+---
+title: Badges
+also:
+---
+{% extends 'component.njk' %}
+{% block content %}
+  <section>
+    <h2 id="link-badges">Link Badges</h2>
+    <div class="demo">
+      <a href="">Link</a><span class="hxBadge">1</span>
+    </div>
+    {% code 'html' %}
+      <a href="">Link</a><span class="hxBadge">1</span>
+    {% endcode %}
+  </section>
+
+  <section>
+    <h2 id="icon-badges">Icon Badges</h2>
+    <div class="demo">
+      <hx-icon type="bell"></hx-icon><span class="hxBadge"></span>
```

Comment:
Did you ever figure out the issue with the extra space when the badge is on a separate line?